mbbsd/register.c

```diff
@@ -70,6 +70,9 @@ register_count_email(const userec_t *u, const char *email);
 static int
 register_check_and_update_emaildb(const userec_t *u, const char *email);
 
+bool
+check_email_allow_reject_lists_core(char *email, const char **errmsg, const char **notice_file, const char *white_email, const char *ban_email);
```

Comment:
You are not using this function before it's definition. Why add a forward declaration?

Reply:
It's just my personal preference (to easily know what are the functions in the file by reading the top of the files / header files.) I'll remove it.

Reply:
actually check_email_allow_reject_lists_core is defined at line: 1021, but is used by check_email_allow_reject_lists at line: 1017.

It's my personal preference to minimize the diff with the original check_email_allow_reject_lists.

I can move check_email_allow_reject_lists after check_email_allow_reject_lists_core to remove the declaration.